src/ray/raylet/node_manager.cc

```diff
@@ -2003,12 +2015,15 @@ void NodeManager::HandleShutdownRaylet(rpc::ShutdownRayletRequest request,
                      "request RPC is ignored.";
     return;
   }
-  auto shutdown_after_reply = []() {
+  auto shutdown_after_reply = [&]() {
     rpc::DrainServerCallExecutor();
     // Note that the callback is posted to the io service after the shutdown GRPC request
     // is replied. Otherwise, the RPC might not be replied to GCS before it shutsdown
     // itself.
-    ShutdownRayletGracefully();
+    rpc::NodeDeathInfo node_death_info;
+    node_death_info.set_reason(rpc::NodeDeathInfo::EXPECTED_TERMINATION);
+    node_death_info.set_reason_message("ShutdownRaylet RPC has been received.");
```

Comment:
This message is not user facing. I think we should say something like `Node is terminated by autoscaler`.